evap/static/scss/evap.scss

```diff
@@ -138,6 +138,10 @@ td.icon {
     padding-left: 4px;
 }
 
+.space-normal {
+    white-space: normal;
```

Comment:
why do we need this?

Reply:
we discussed that the line breaks in the email form buttons would be strange otherwise.